src/managers/HookManager.sol

```diff
@@ -109,23 +109,13 @@ abstract contract HookManager is IHookManager, Auth {
   }
 
   // Runs the validation hooks that are enabled by the account and returns true if none reverts
-  function runValidationHooks(
-    bytes32 signedHash,
-    Transaction calldata transaction,
-    bytes[] memory hookData
-  ) internal returns (bool) {
+  function runValidationHooks(bytes32 signedHash, Transaction calldata transaction) internal returns (bool) {
     mapping(address => address) storage validationHooks = _validationHooksLinkedList();
 
     address cursor = validationHooks[AddressLinkedList.SENTINEL_ADDRESS];
-    uint256 idx;
     // Iterate through hooks
     while (cursor > AddressLinkedList.SENTINEL_ADDRESS) {
-      // Call it with corresponding hookData
-      bool success = _call(
```

Comment:
This was what was returning transaction reverts into booleans for the validation return, which we had to move into the validation module itself for expected failures.